README.md

```diff
@@ -4,12 +4,38 @@
 
 We are the @nodejs/automation team.
 
-## Purpose
+## About the Team
 
 The Node.js Automation Team focuses on building tools, bots and scripts that
 help Node.js collaborators maintain the Node.js project.
 
-## Projects
+The automation team is currently structured like this:
 
-To be discussed.
+- **@nodejs/automation**: people to ping for automation-related issues in
+the organization.
+  - **@nodejs/automation-collaborators**: has write access to all
+  automation-related repositories.
+    - **@nodejs/automation-admins**: has admin access to all
+    automation-related repositories.
 
+## Projects / Repositories
```

Comment:
needs `lts` tool as well

Reply:
Do you mean https://github.com/continuationlabs/lts ? It has not been transferred into the foundation yet..

Reply:
I guess we will wait for https://github.com/nodejs/Release/pull/277 and for it to be transferred (need an issue in the TSC repo first?), we probably should not delay this PR for that, can add it later once it's been transferred.

Reply:
Yep that's the one -- I thought it had already been transferred to the nodejs org... cc @cjihrig

Reply:
https://github.com/nodejs/TSC/issues/410